lib/parsers/index.ts

```diff
@@ -22,6 +17,18 @@ export enum DepType {
   dev = 'dev',
 }
 
+export interface ReferenceInclude {
+  Version?: string;
+  Culture?: string;
+  processorArchitecture?: string;
+  PublicKeyToken?: string;
+}
+
+export interface DependenciesDiscoveryResult {
```

Comment:
Do we need to export this interface? In addition, do we need to export `getDependenciesFromReferenceInclude` and `getDependenciesFromPackageReference`?

Reply:
we don't i though i might want to test it separately